src/screens/WyreWebview.js

```diff
@@ -19,15 +19,16 @@ const StyledWebView = styled(WebView)`
   background-color: ${colors.white};
 `;
 const HeaderTitle = styled(Text).attrs({
-  align: 'left',
+  align: 'center',
```

Comment:
Instead of making it 100% width and aligning content, please use `headerTitleAlign: center` in `screenOptions` in `Stack.Navigator`.